src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -286,7 +301,19 @@ private String returnTypeComment(
     contentBuilder.append(
         "\n\nRaises:\n  :exc:`google.gax.errors.GaxError` if the RPC is aborted.");
 
-    return Splitter.on("\n").splitToList(contentBuilder.toString());
+    return splitToLines(contentBuilder.toString());
+  }
+
+  public List<String> enumValueComment(EnumValue value) {
+    String description = "";
+    if (value.hasAttribute(ElementDocumentationAttribute.KEY)) {
+      String sphinxified = getSphinxifiedScopedDescription(value);
+      description = sphinxified.replaceAll("\\s*\\n\\s*", "\n");
```

Comment:
NIT: This is duplicated above. It should be broken out into a private method, but I'm not going to block 0.10.0 for this.